src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -83,23 +112,49 @@ export default class DataTable extends Vue {
     let cellIndex = 0;
 
     Object.keys(this.data.head).forEach((column: string) => {
-      const label = this.data.head[column].label || this.data.head[column],
+      const infoPopoverId = `info-popover-${dataTableNumber}-${column}`,
+        label = this.data.head[column].label || this.data.head[column],
+        infoIcon = this.data.head[column].description ? (
+          <chi-button
+            id={infoPopoverId}
+            variant="flat"
+            type="icon"
+            alternative-text="Info icon"
+            onclick={() => {
+              this._toggleInfoPopover(infoPopoverId);
+            }}>
+            <i class={`${ICON_CLASS} ${INFO_ICON_CLASS}`} aria-hidden="true"></i>
+          </chi-button>
+        ) : null,
+        infoPopover = this.data.head[column].description ? (
+          <chi-popover
+            ref={infoPopoverId}
+            reference={`#${infoPopoverId}`}
+            position="top"
+            title={(this.data.head[column].description as DataTableColumnDescription).title}
+            portal={this._chiMajorVersion === 4}
+            modal={this._chiMajorVersion === 5}>
+            <div>{this._getDescription(this.data.head[column].description as string | DataTableColumnDescription)}</div>
+          </chi-popover>
+        ) : null,
         sortBy = this.data.head[column].sortBy,
         sortable = this.data.head[column].sortable,
         alignment = this._cellAlignment(this.data.head[column].align || 'left'),
         sortIcon = sortable ? (
-          <i
-            class={`
+          <chi-button variant="flat" type="icon" alternative-text="Sort icon">
```

Comment:
The sorting is broken. Clicking the column label or blank space of the cell does not trigger sorting.
Stree-test the changes, please.

https://user-images.githubusercontent.com/25136131/148736989-29a45caf-2cdc-415d-8bd9-f6104d322a74.mov

Reply:
i'm investigating this because on local it works perfectly 🤔

Reply:
Fixed!